static/css/styles.css

```diff
@@ -680,7 +687,7 @@ button.menu-button + nav a {
 }
 
 @media screen and (min-width: 750px) {
-    .profile-page main {
+    .profile-page main #content {
         display: grid;
```

Comment:
You are right about mixing grid and non-grid styles but what's the point to use it when it comes to 1 column centered ? I guess it is just to act strict and rigorous, which I ask myself if it worth the shot ? Let's talk about it tonight. I merge the PR, but keep in mind it can be changed later

Reply:
It's more about organizing the page withing the content than strictness. I didn't mean to put a grid just for a centering ;)

Reply:
ok, I'll try then